.dockerignore

```diff
@@ -1,2 +1,10 @@
 .virtualenv
 .venv
+.mypy_cache
+.pytest_cache
+.git
+build
```

Comment:
I don't think we use this directory.

Reply:
I added disk usage command's output above from the latest image, there's `build` directory. 
If we don't use it - it won't hurt the docker image.